examples/whisper.android.java/app/src/main/jni/whisper/CMakeLists.txt

```diff
@@ -6,36 +6,25 @@ set(CMAKE_CXX_STANDARD 17)
 set(WHISPER_LIB_DIR ${CMAKE_SOURCE_DIR}/../../../../../../../)
 
 set(SOURCE_FILES
-    ${WHISPER_LIB_DIR}/ggml/src/ggml.c
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/ggml-cpu.c
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/traits.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/arch/arm/quants.c
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/arch/arm/repack.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/quants.c
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/ggml-cpu.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/unary-ops.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/binary-ops.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/vec.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-cpu/ops.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-alloc.c
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-backend.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-backend-reg.cpp
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-quants.c
-    ${WHISPER_LIB_DIR}/ggml/src/ggml-threading.cpp
     ${WHISPER_LIB_DIR}/src/whisper.cpp
     ${CMAKE_SOURCE_DIR}/jni.c
     )
 
 find_library(LOG_LIB log)
 
+include(FetchContent)
+
 function(build_library target_name)
     add_library(
         ${target_name}
         SHARED
         ${SOURCE_FILES}
     )
 
-    target_link_libraries(${target_name} ${LOG_LIB} android)
+    FetchContent_Declare(ggml SOURCE_DIR ${WHISPER_LIB_DIR}/ggml)
+    FetchContent_MakeAvailable(ggml)
```

Comment:
Nice! I didn't know that it is possible to use a local source like this.

Reply:
Neither did I, but I was nice to discover that this is possible :)